plugins/labhub.py

```diff
@@ -13,6 +13,12 @@
 from utils.mixin import DefaultConfigMixin
 
 
+def validate_nick(msg):
```

Comment:
I fail to understand why you are doing this. This function merely returns `bool(msg.frm.nick)`. You can just use `if msg.frm.nick:` instead of `if validate_nick(msg)`.. This function does no good.